app/src/main/java/org/dhis2/utils/analytics/matomo/TrackerController.kt

```diff
@@ -0,0 +1,34 @@
+package org.dhis2.utils.analytics.matomo
+
+import org.dhis2.BuildConfig
+import org.matomo.sdk.Matomo
+import org.matomo.sdk.Tracker
+import org.matomo.sdk.TrackerBuilder
+
+class TrackerController {
+    companion object {
+        fun generateTracker(matomo: Matomo): Tracker? {
+            return when (BuildConfig.DEBUG) {
+                true -> null
+                false -> TrackerBuilder.createDefault(
+                    BuildConfig.MATOMO_URL,
+                    BuildConfig.MATOMO_ID
+                ).build(matomo)
+            }
+        }
+
+        fun generateSecondaryTracker(
```

Comment:
can you rename these trackers? For example: dhis2InternalTracker, dhis2ExternalTracker ?